wa/workloads/facebook/__init__.py

```diff
@@ -0,0 +1,44 @@
+#    Copyright 2013-2015 ARM Limited
```

Comment:
The commit message references the wrong issue, it should be https://github.com/ARM-software/workload-automation/issues/704